.github/workflows/auto-comment.yml

```diff
@@ -1,5 +1,9 @@
 name: Auto Comment
-on: [issues, pull_request]
+on:
+  pull_request:
+    paths:
```

Comment:
Oh this is cool, I didn't know you could do this. Is there a reason it's set up for any changes in the ``prisma`` directory and not only when the ``schema.prisma`` file changes?

Reply:
no particular reason. I can change it to just that file if you prefer

Reply:
done